src/verifiable-presentation.js

```diff
@@ -8,6 +8,7 @@ import {
 } from './utils/type-helpers';
```

Comment:
The changes made in this file are same as the previous one? Can use abstract this behavior in separate functions like `addUniqueContext`, `addUniqueType` and then use them at both places

Reply:
Thanks! Maybe a superclass would be even cleaner, feels like an overkill atm though.

Reply:
I wasn't implying a superclass, an independent util function taking the mutable array would work. No big deal.